docs/02-app/02-api-reference/06-create-next-app.mdx

```diff
@@ -26,7 +26,7 @@ pnpm create next-app
 ```
 
 ```bash filename="Terminal"
-bunx create-next-app
+bunx create next-app
```

Comment:
Looks like according to the docs these are equivalent https://bun.sh/docs/cli/bun-create#from-npm

Reply:
![image](https://github.com/vercel/next.js/assets/50093149/4e1071cf-6a76-45e8-81c5-e3a728cb0908)

it throws a error

Reply:
but works well when it is bunx create next-app

Reply:
Hmm might want to report upstream but we can land if this fixes since it should be the same